handbook/engineering/developer-insights/index.md

```diff
@@ -44,6 +44,28 @@ _Updated 2020-03-08_
 The strategy is currently a work in progress and can be viewed here: [Developer Insights FY22 Strategy](https://docs.google.com/document/d/18GS2Gr7SP2ICuJaOpofxiwuxU3pKBfp8eXREFLDcM30/edit)
 _(Only accessible to Sourcegraph teammates until finalized.)_
 
+## Principles and practices
+
+In addition to the [engineering principles and practices](../principles-and-practices.md) the Developer Insights org follows the following principles and practices.
+
+### Unblocking a teammate is your highest priority
+
+If a teammate is blocked by you on a question, your approval, or a pull request review, your top priority is always to unblock them, either directly or through helping them find someone else who can, even if this takes time away from your own or your team's priorities.
```

Comment:
Question: How do we define 'blocked'? Is it more of a case of the teammate making it clear that something needs to be resolved for them to continue, or does this include, for example, general PRs where I might need my teammates review to continue.

Reply:
@pdubroy yes I agree with your sentiment that waiting for a PR review might not block you. One can make the case that it blocks overall progress, but I don't want teammates to feel they need to drop everything for a PR review, especially if they can communicate and manage expectations with the PR author as described further down.

Should we update the wording to make this more explicit?

Reply:
I've gone ahead and merged this PR so that I can link to the principles, but will create a follow-up PR with improved wording.